json-rpc/types/src/views.rs

```diff
@@ -609,9 +609,6 @@ impl MetadataView {
             self.script_hash_allow_list = Some(vm_publishing_option.script_allow_list);
             self.module_publishing_allowed = Some(vm_publishing_option.is_open_module);
         }
-        if let Some(diem_version) = diem_root.get_diem_version()? {
-            self.diem_version = Some(diem_version.major);
-        }
```

Comment:
this is a breaking change, do we really need remove it?

Reply:
and looks like you added similar logic to client:
```
        if let Some(diem_version) = diem_root.get_diem_version().map_err(Error::rpc_response)? {
            metadata_view.diem_version = Some(diem_version.major);
        }
```

then why we remove this from server side?

Reply:
and looks like you added similar logic to client:
```
        if let Some(diem_version) = diem_root.get_diem_version().map_err(Error::rpc_response)? {
            metadata_view.diem_version = Some(diem_version.major);
        }
```

then why we remove this from server side?